modules/control-utility/src/test/java/org/apache/ignite/util/KillCommandsCommandShTest.java

```diff
@@ -271,16 +287,25 @@ public void testCancelConsistencyTask() throws InterruptedException {
             thLatch.countDown();
         });
 
+        // GridNearGetRequest messages count required to pefrom getAll() with readRepair from all nodes twice.
+        // First will be finished (which generates status), second will be frozen.
+        int twiceGetMsgCnt = SERVER_NODE_CNT * (SERVER_NODE_CNT - 1) * 2;
```

Comment:
Looks like this test can be simplified. If the consistency status map will be notified on the repair task just started then we just "0/10000" instead of "1024/10000"). So, there is no need to implement complex logic for waiting specific events? WDYT?

Reply:
0/10000 will not show real progress, just start.
While 1024/10000 (1024 will be defined as a const later) means counter works.